style.css

```diff
@@ -0,0 +1,157 @@
+@import url(https://fonts.googleapis.com/css?family=Roboto);
+
+* {
+  font-family: "Roboto";
+  margin: 5px;
+}
+
+.bottom_border {
+  border-bottom: thin solid lightgray;
+}
+
+nav > ul {
+  display: grid;
+  width: 90%;
+  height: 75px;
+  grid-template-columns: 80% 15% 5%;
+  list-style: none;
+  font-weight: bold;
+  font-size: 85%;
+}
+
+.heading {
+  height: 10em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding-bottom: 5em;
+  background-color: lavender;
+}
+
+#sub {
+  font-size: 1em;
+  text-transform: capitalize;
+  font-weight: bold;
+}
+
+#action {
+  border-radius: 10px;
+  border: solid lightgray;
+  padding: 1em;
+  margin: 2em;
+  text-decoration: none;
+  color: black;
+}
+
```

Comment:
The above section is formatted well.  Nice spacing, I like the choice of formatting for the tagline.